docs/sections/user_guide/cli/tools/execute.rst

```diff
@@ -0,0 +1,58 @@
+``execute``
+===========
+
+The ``uw`` mode for executing external drivers.
+
+.. literalinclude:: execute/help.cmd
+   :language: text
+   :emphasize-lines: 1
+.. literalinclude:: execute/help.out
+   :language: text
+
+.. _cli_execute_examples:
+              
+Examples
+^^^^^^^^
+
+These examples use the following inputs:
+
+Module ``rand.py``
+
+.. literalinclude:: execute/rand.py
+   :language: python
+
+Schema ``rand.jsonschema``
+
+.. literalinclude:: execute/rand.jsonschema
+     :language: json
+
+Config ``rand.yaml``
+
+.. literalinclude:: execute/rand.yaml
+   :language: yaml
+
+* Execute the external driver:
+
+  .. literalinclude:: execute/execute.cmd
+     :language: text
+     :emphasize-lines: 2
+  .. literalinclude:: execute/execute.out
+     :language: text
+  
+* If the external driver does not accept an argument that was provided on the command line, it will exit with error. In this case, ``Rand`` inherits from parent class ``AssetsTimeInvariant``, which does not accept a ``cycle`` argument:
+
+  .. literalinclude:: execute/bad-arg.cmd
+     :language: text
+     :emphasize-lines: 1
+  .. literalinclude:: execute/bad-arg.out
+     :language: text
+  
+* If the schema file for a driver resides in the same directory as its Python module and has the same filename prefix, as well as a ``.jsonschema`` suffix (e.g. ``rand.jsonschema`` alongside ``rand.py``) then the ``--schema-file`` argument is not required. However, ``--schema-file`` can be used to point to an alternate schema:
+
+  .. literalinclude:: execute/alt-schema.cmd
+     :language: text
+     :emphasize-lines: 2
+  .. literalinclude:: execute/alt-schema.out
+     :language: text
+
+* Other arguments behave identically to the same-named arguments to built-in ``uwtools`` drivers (see :ref:`drivers`).
```

Comment:
Do you think it's worth explicitly calling out what qualifies as a "task" on this page? For example, if a user tries to use the flag `--task driver_name` (an attribute) it fails:

```
Traceback (most recent call last):
  File "/Users/christina.holt/conda/envs/DEV-uwtools/bin/uw", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/christina.holt/work/uwtools/src/uwtools/cli.py", line 91, in main
    sys.exit(0 if modes[args[STR.mode]](args) else 1)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/christina.holt/work/uwtools/src/uwtools/cli.py", line 291, in _dispatch_execute
    return uwtools.api.driver.execute(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/christina.holt/work/uwtools/src/uwtools/api/driver.py", line 89, in execute
    getattr(driverobj, task)()
TypeError: 'str' object is not callable
``` 

Maybe it's as simple as a sentence about which decorators elevate a function to a "task".

Reply:
Good idea, will do.